client/src/components/Navbar/Navbar.tsx

```diff
@@ -155,7 +174,12 @@ const Navbar: React.FC = () => {
                     onClick={handleMenuOpen}
                     color="inherit"
                   >
-                    <img style={{ width: 50 }} src={`https://robohash.org/${loggedInUser.email}`} />
+                    <img
+                      key={Date.now()}
+                      style={{ width: 50, height: 50, borderRadius: '50%', objectFit: 'cover' }}
```

Comment:
Please avoid the usage of inline styles, lets resort to using a CSS class to apply these styles instead